index.html

```diff
@@ -1133,6 +1133,55 @@ <h3>Revocation Bitstring Length</h3>
       </p>
     </section>
 
+    <section class="informative">
+      <h3>Unnecessary Correlation</h3>
+
+      <p>
+There are a number of global identifiers used in a status list entry, defined in
+Section [[[#bitstringstatuslistentry]]], that can be used across [=verifiers=]
+to correlate [=subjects=]. Some of these properties include `id`,
+`statusListIndex`, and `statusListCredential`.
+      </p>
+
+      <p>
+In some cases, when [=presenting=] a [=verifiable credential=] that contains
+other global identifiers (such as a driver's license identification number),
+using other global identifiers for the status list information does not cause
+further correlation harms since the only thing required for correlation is a
+single globally unique identifier.
+      </p>
+
+      <p>
+When these global identifiers are used in [=presentations=] that utilize
+[=selective disclosure=] or [=unlinkable disclosure=], they can violate privacy
+expectations. [=Issuers=] are urged to enable status information to be
+selectively disclosable when there is an expectation that a particular
+[=verifiable credential=] can be disclosed in a way that avoids
+correlation, such as proving that an individual is above a certain age.
+[=Verifiers=] can require that status information is revealed in situations that
+require them to know the current status of a credential, and the holder might
+then consent to revealing that information or might refuse to do so for a given
+transaction. In all cases, both [=issuers=] and [=verifiers=] are urged to
+ensure that the use of global identifiers is avoided when a particular exchange
```

Comment:
@David-Chadwick,

I'd suggest this instead:

```suggestion
ensure that the use of global identifiers can be avoided when a particular exchange
```

I think this should cover "enabling" holders to make the selective disclosure choices they'd prefer to make when future exchanges arise where the global identifiers don't need to be disclosed.

Reply:
The text that I object to is 

> In all cases, .. [=issuers=] ..  are urged to
> ensure that the use of global identifiers is avoided when a particular exchange
> does not require correlation.

the reason being that issuers do not know what exchanges the holder may participate in, and therefore which might or might not require correlation

Reply:
Whilst I agree with what you say above, the sentence I object to does not reflect this. So can we edit the sentence please so that it does reflect this e.g.

> In all cases, [=verifiers=] are urged to
> ensure that the use of global identifiers is avoided when a particular exchange
> does not require correlation. Similarly [=issuers=] that support selective disclosure 
> should make everything selectively disclosable by default in order to help the holder 
> maximize their privacy in future exchanges.

Please edit the above as you wish